### PR TITLE
Assignment-4 modified Test Cases

### PR DESCRIPTION
**Pull request by mayankrajendrat**

### CODE REVIEW

File: src/Assignment-4/assignment4_test.js [hunk 1]
```diff
@@ -14,7 +14,7 @@
   var quixote = require("../../vendor/quixote.js");
 
   // Define a test suite using Mocha's standard `define` function.
-  describe("Assignment-4", function () {
+  describe.only("Assignment-4", function () {
```
Comment (sailesh-ramayanam) [inline src/Assignment-4/assignment4_test.js:17]: Why do we require `.only` here?

File: src/Assignment-4/assignment4_test.js [hunk 2]
```diff
@@ -64,16 +64,9 @@
...
       assert.isTrue(
-        isExist(
-          frame.toDomElement().contentDocument.getElementsByTagName("img")
-        )
-      );
-    });
-    it("input should have name attribute", function () {
-      assert.isTrue(
-        frameDom.getElementsByTagName("input")[0].hasAttribute("name")
+          frame.toDomElement().contentDocument.querySelectorAll("header nav ul")[0].childElementCount=== 3
```
Comment (sailesh-ramayanam) [inline src/Assignment-4/assignment4_test.js:69]: Assignment-4 has already been given to students and it is completed. Why are we modifying these test cases?

**Comment by sailesh-ramayanam** [comment]
@satya-9 Please review `HTML_CSS Assignment-5.docx`.

**Comment by sailesh-ramayanam** [comment]
@mayankrajendrat Do we have test cases for Assignment-5 ?

**Comment by satya-9** [comment]
> @satya-9 Please review `HTML_CSS Assignment-5.docx`.

HTML_CSS Assignment-5.docx  is fine.

File: src/Assignment-4/assignment4_test.js (hunk 1 above)
Comment (mayankrajendrat) [reply src/Assignment-4/assignment4_test.js:17]: As per the instruction forwarded in group for Step 2 in "Running test cases"
https://the10x-academy.slack.com/archives/C027J47DQ12/p1637067590217500.

File: src/Assignment-4/assignment4_test.js (hunk 2 above)
Comment (mayankrajendrat) [reply src/Assignment-4/assignment4_test.js:69]: Some of the students faced generic issues related to tag counts and classes, so we made them more precise.

**Comment by mayankrajendrat** [comment]
> @mayankrajendrat Do we have test cases for Assignment-5 ?

yes, in place.

File: src/Assignment-4/assignment4_test.js (hunk 1 above)
Comment (sailesh-ramayanam) [reply src/Assignment-4/assignment4_test.js:17]: That was a temporary workaround to enable the students to be able to run only a particular set of test cases. Let us not have `.only` in the repo. Please remove it.

**Review by sailesh-ramayanam — APPROVED** [review]

**Pull request merged**